src/common/testing/BUILD

```diff
@@ -1,3 +1,20 @@
+#-------------------------------------------------------------------------------
+# Copyright 2022 Google LLC
```

Comment:
Again, is this 2022? testing is one of the first things we added, but maybe this `BUILD` file came from a code reorg.